README.md

```diff
@@ -1,136 +1,79 @@
-# We are Nmstate!
+# Declarative Network API
```

Comment:
Why we are not Nmstate anymore? :(

I thought this gave us some personality and it was friendly!

Reply:
OK. Let me revert it.